lib/diagnostic.rb

```diff
@@ -4,23 +4,25 @@
 # Running scripts from the command line
 #
 # In a Ruby string, write the command you would use to run a script at
+
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = pry './lib/example.rb'
 
```

Comment:
Technically this would go back one folder. So if we're in project, and want to access `project/lib/example.rb` we only need `lib/example.rb` this would go up one parent directory first.